code/__DEFINES/mobs.dm

```diff
@@ -548,6 +548,26 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define RUNNER_EVASION_RUN_DELAY			0.5 SECONDS //If the time since the Runner last moved is equal to or greater than this, its Evasion ends.
 #define RUNNER_EVASION_DANGER_RATIO			0.5 //If we have this % of auto-dodge damage remaining or less, warn the user.
 
+//Wraith defines
+
+#define WRAITH_PLACE_WARP_BEACON_WINDUP		3 SECONDS
+
+#define WRAITH_HYPERPOSITION_MIN_WINDUP		0.5 SECONDS
+#define WRAITH_HYPERPOSITION_MAX_WINDUP		5 SECONDS
+
+#define WRAITH_PHASE_SHIFT_WINDUP			1 SECONDS
+#define WRAITH_PHASE_SHIFT_DURATION			5 SECONDS
+#define WRAITH_PHASE_SHIFT_DURATION_WARNING	0.7
+#define WRAITH_PHASE_SHIFT_ALPHA			128 //50% transparency
+
+#define WRAITH_BLINK_DRAG_MULTIPLIER		20 //The amount we multiply the cooldown by when we teleport while dragging a target
+#define WRAITH_BLINK_RANGE					3
+
+#define WRAITH_BANISH_BASE_DURATION			10 SECONDS
```

Comment:
Thinking on this some more since we discussed it, it's probably best to keep it as is, especially with the portal sfx duration relying on this define.

Reply:
I assume that this comment may have stemmed from the initial understanding that a marine would be banished for 10 seconds as opposed to 5 in actuality.